library.js

```diff
@@ -0,0 +1,211 @@
+        parsedVersion.patch,
+      ].join('.'));
+    } else {
+      reject(new Error('Unable to get local NW.js version'));
+    }
+  });
+}
+
+function getCorrectNodeVersion () {
+  return new Promise(async (resolve, reject) => {
+    try {
+      const localNwVersion = await getLocalNWVersion();
+      const allNwVersions = await getVersions();
+
+      const match = allNwVersions.versions.find(function (version) {
+        // 'v0.82.0' === 'v' + '0.82.0'
+        return version.version === 'v' + localNwVersion;
+      });
+      if (!match) {
+        throw 'No matching NW.js version found';
+      }
+      const nodeVersion = match?.components?.node;
+      if (nodeVersion) {
+        resolve(nodeVersion);
+      } else {
+        throw 'No Node version found';
+      }
+    } catch (error) {
+      reject(error);
+    }
+  });
+}
+
+function determineOriginalManifestIndentation (data) {
+  data = data.trim();
+  data = data.replaceAll('\r\n', '\n');
+
+  if (data[0] !== '{' || data[1] !== '\n') {
+    return;
+  }
+
+  let first = data[2];
+  let second = data[3];
+  let third = data[4];
+  let fourth = data[5];
+
+  if (first === '\t') {
+    originalManifestIndentation = '\t';
+  } else if (first + second + third + fourth === '    ') {
+    originalManifestIndentation = 4;
+  } else {
+    originalManifestIndentation = 2;
+  }
+}
+
+function determinOriginalEOL (data) {
+  if (data.includes('\r\n')) {
+    originalManifestEOL = '\r\n';
+  } else {
+    originalManifestEOL = '\n';
+  }
+}
+
+function getManifestPath () {
+  return new Promise(async (resolve, reject) => {
+    const manifestRelativeToCwd = path.resolve(process.cwd(), 'package.json');
+    const manifestRelativeToHere = path.resolve(__dirname, '..', '..', 'package.json');
+    const cwdExists = await fileExists(manifestRelativeToCwd);
+    const hereExists = await fileExists(manifestRelativeToHere);
+    if (cwdExists) {
+      resolve(manifestRelativeToCwd);
+    } else if (hereExists) {
+      resolve(manifestRelativeToHere);
+    } else {
+      reject(new Error('Could not locate your manifest.'));
+    }
+  });
+}
+
+function getManifest () {
+  return new Promise(async (resolve, reject) => {
+    try {
+      const manifest = await getManifestPath();
+      let data = await fs.promises.readFile(manifest, { encoding: 'binary' });
+      determineOriginalManifestIndentation(String(data));
+      determinOriginalEOL(String(data));
+      data = JSON.parse(data);
+      resolve(data);
+    } catch (error) {
+      reject(error);
+    }
+  });
+}
+
+function getManifestWithUpdatedVoltaObject () {
+  return new Promise(async (resolve, reject) => {
+    try {
+      const manifest = await getManifest();
+      const nodeVersion = await getCorrectNodeVersion();
+      manifest.volta = manifest.volta || {};
+      manifest.volta.node = nodeVersion;
+      resolve(manifest);
+    } catch (error) {
+      reject(error);
+    }
+  });
+}
+
+export async function run () {
```

Comment:
I renamed the `index.js` to `library.js`, added an `export` to this line and removed the `run()`.

Those are the only changes here. I don't know why git didn't see this as a rename.